vignettes/tidytranscriptomics.Rmd

```diff
@@ -1084,6 +1089,11 @@ pbmc_small_nested_reanalysed <-
 pbmc_small_nested_reanalysed
 ```
 
+## Key Points
+- Some basic steps of a single-cell RNA sequencing analysis are dimensionality reduction, cluster identification and cell type classification
+- `tidyseurat` is a invisible layer that operates on `Seurat` object and allows to visualise and manipulate data as if it were a tidy data frame.
+- `tidyseurat` object are `Seurat object` so they can be used with any `Seurat` compatible method
```

Comment:
```suggestion
- `tidyseurat` object is a `Seurat object` so it can be used with any `Seurat` compatible method
```

Reply:
Is not accepting

Reply:
that is weird, are you able to add a message into the Commit suggestion dropdown like in the gif here: https://github.com/sindresorhus/refined-github/issues/2122#issuecomment-499011026
do you get a permission denied error like in that gif?

Reply:
I click on commit suggestion and the text does not appear to change. However feel free to accept everything and merge